internal/server/config/config_test.go

```diff
@@ -250,6 +250,13 @@ func TestVerifyConfig(t *testing.T) {
 func TestDefaultMaxConditionValuationCost(t *testing.T) {
 	// check to make sure DefaultMaxConditionEvaluationCost never drops below an explicit 100, because
 	// API compatibility can be impacted otherwise
+	t.Run("max_condition_evaluation_cost_too_low", func(t *testing.T) {
```

Comment:
Please also add a test where max condition evaluation cost is above the default.